app-backend/db/src/test/scala/com/azavea/rf/database/AnnotationProjectDaoSpec.scala

```diff
@@ -302,6 +302,16 @@ class AnnotationProjectDaoSpec
             "Readiness was updated"
           )
 
+          assert(
+            afterUpdate.campaignId == annotationProjectUpdate.campaignId,
```

Comment:
I think this technically isn't testing anything, since the test would still pass if the logic to update the campaign ID were removed.. The generator for `AnnotationProject.Create`s [doesn't include](https://github.com/raster-foundry/raster-foundry/blob/24b7304250c3e06d284a9fb0f17f32c9cc52cb04/app-backend/common/src/test/scala/com/implicits/Generators.scala#L1106) a campaign id, so both of them start with `None` and should end with `None`.